src/plots/cartesian/dragbox.js

```diff
@@ -543,8 +548,12 @@ module.exports = function dragBox(gd, plotinfo, x, y, w, h, ns, ew) {
             pushActiveAxIds(yaLinked);
         }
 
+        updates = {};
         for(i = 0; i < activeAxIds.length; i++) {
-            doTicks(gd, activeAxIds[i], true);
+            var axId = activeAxIds[i];
+            doTicks(gd, axId, true);
+            var ax = getFromId(gd, axId);
+            updates[ax._name + '.range'] = ax.range.slice();
```

Comment:
-> back to independent attributes in https://github.com/plotly/plotly.js/pull/1767/commits/51fc2c694b4e89055bfb9a02e392d1ca41e4ed03